### ISSUE TITLE
Test performance of multi subdomain register solution

### ISSUE THREAD

**Issue by frelun** [Registercentrum/stratum-cms#18]

**Comment by frelun** [comment]
Has been carrying out load tests with help from JMeter. Current tests more or less makes static requests to a randomized set of register pages in parallel. 
![graph](https://cloud.githubusercontent.com/assets/3897250/8327970/f236159a-1a6c-11e5-92b6-39173f365f70.png)

**Comment by frelun** [comment]
Closing this for now, open a new issue further in when this gets more crucial
